src/concrete/ml/sklearn/base.py

```diff
@@ -2069,22 +2106,39 @@ def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
         Returns:
             numpy.ndarray: The majority vote.
         """
-        y_preds_processed = []
-        for y in y_preds:
-            vote = self.majority_vote(y.flatten())
-            y_preds_processed.append(vote)
+        return numpy.array([self.majority_vote(y.flatten()) for y in y_preds])
```

Comment:
not sure why you have made this change, it's a bit harder to follow with one-liner (especially without any comments) !

Reply:
The comment is in the docsting, for me it was obvious...
shall I get it back ?